### DISCUSSION TITLE
Show summary of border flow

### DISCUSSION THREAD

**Discussion by DerDakon** [electricitymaps/electricitymaps-contrib#8343] [Ideas]
I would welcome if the border flow would have a header showing something like "total: 0,2GW import / 1,7 GW export: sum 1,5GW export". Bonus if the import/export section can be collapsed and shows only the summary.

**Comment by VIKTORVAV99** [comment]
I'm going to move this to a discussion as we are trying to clean up the issues by only including actionable problems.
